stable/artifactory-ha/values.yaml

```diff
@@ -1064,6 +1066,18 @@ nginx:
       if ($http_x_forwarded_proto = '') {
         set $http_x_forwarded_proto  $scheme;
       }
+      {{- if .Values.nginx.service.ssloffload }}
+      if ($host = $server_addr) {
+        rewrite ^/artifactory/webapp/(.*) http://$host:8081/artifactory/webapp/$1 redirect;
```

Comment:
Change 8081 to `{{ .Values.artifactory.externalPort }}`

Reply:
$server_addr is the address of the localhost in which Nginx runs. This is to make sure the Nginx readinessProbe/livenessProbe doesn't break. Nginx checks for artifactory health in https protocol. Another server block to listen in 8081 was configured which redirects to the application.

Reply:
@danielezer Suggest a better way to handle the Nginx readiness/liveness